test/libsolidity/SemanticTest.cpp

```diff
@@ -271,9 +287,50 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 		return TestResult::Failure;
 	}
 
+	if (m_gasCostFailure)
+	{
+		AnsiColorized(_stream, _formatted, {BOLD, CYAN})
+			<< _linePrefix << "Gas results missing or wrong, obtained result:" << endl;
+		for (auto const& test: m_tests)
+		{
+			ErrorReporter errorReporter;
+			_stream << test.format(errorReporter, _linePrefix, false, _formatted) << endl;
```

Comment:
What does it print here? Would be nice to see:
 - old value
 - new value
 - absolute change
 - relative change

Reply:
I will modify the output when tests are failing definitely